core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -85,16 +87,17 @@ abstract contract FeePayer is Testable, Lockable {
 
     /**
      * @notice Pays UMA DVM regular fees (as a % of the collateral pool) to the Store contract.
-     * @dev These must be paid periodically for the life of the contract. If the contract has not paid its
-     * regular fee in a week or more then a late penalty is applied which is sent to the caller.
-     * This will revert if the amount of fees owed are greater than the pfc. An event is only fired if the fees charged are greater than 0.
+     * @dev These must be paid periodically for the life of the contract. If the contract has not paid its regular fee
+     * in a week or more then a late penalty is applied which is sent to the caller. This will revert if the amount of
+     * fees owed are greater than the pfc. An event is only fired if the fees charged are greater than 0.
      * @return totalPaid Amount of collateral that the contract paid (sum of the amount paid to the Store and caller).
```

Comment:
Remove `totalPaid` if you're not going to name it

Reply:
Actually, across the board, we've kept the name of the variable returned in the Natspec, even if we return a named value. Check the DVM natspec implementation for example.

However, It does suck that this we cant make this change in all places. What's your thoughts on this inconsistency?

Reply:
Look at `retrieveRewards` vs `hasPrice` or `getPrice`. There's inconsistency but we can't easily modify those contracts now.

The EMP is pretty inconsistent, look at `redeem`/`settleExpired` versus `getCollateral` for example. Let's be consistent while we can easily modify these contracts.

I see no reason to pick naming versus not-naming however. @mrice32 any bias?

Reply:
I'm biased towards whichever we do *more often* already, so we're as close to consistent as possible.